src/emulator/auth/widget_ui.ts

```diff
@@ -294,21 +327,67 @@ const STYLE = `
   color: #858585;
 }
 
-body {
-  font-family: "Roboto", sans-serif;
-  margin: 0;
-  padding: 0;
-  width: 100%;
+.subtitle {
+ color: var(--mdc-theme-text-secondary-on-background);
+ font-size: 14px;
+ line-height: 20px;
+ margin-block-end: 0em;
+ margin-block-start: 0em;
 }
 
 #content {
   box-sizing: border-box;
-  padding: 12px;
-  width: 500px;
+  width: 515px;
+  margin: 16px auto;
 }
 
-button {
-  text-transform: none !important;
+.content-wrapper {
+  padding: 0 24px;
+}
+
+.mdc-list--avatar-list .mdc-list-item {
+  padding: 0 24px;
+}
+
+.mdc-list--avatar-list .mdc-list-item__graphic {
+  border-radius: 50%;
+  height: 36px;
+  margin-left: 0;
+  margin-right: 16px;
+  width: 36px;
+}
+
+.mdc-list .mdc-list-item__graphic {
+  display: -ms-inline-flexbox;
+  display: inline-flex;
+}
+
+.mdc-list--avatar-list .mdc-list-item__graphic {
+  background-color: rgba(0,0,0,.3);
+  color: #fff;
+}
+
+.mdc-list-item__graphic {
+  align-items: center;
+  color: background-color: rgba(0,0,0,.3);
```

Comment:
nit/optional: consider using a var?

Reply:
We're not really using variables, but I'm gonna go with an opaque gray.